onboarding-scripts/Makefile

```diff
@@ -0,0 +1,49 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=onboarding-scripts
+PKG_VERSION:=v0.1.0
```

Comment:
It seems that very few packages have PKG_VERSION defined as v[number]. I suggest
```
PKG_VERSION:=0.1.0
...
PKG_SOURCE_VERSION:=v$(PKG_VERSION)
```